lib/invalid_dependency_error.js

```diff
@@ -0,0 +1,21 @@
+var util = require('util')
+
+function InvalidDependencyError (dependency, requiredVersion, installedVersion) {
+  // Super constructor
+  Error.call(this)
+
+  // Super helper method to include stack trace in error object
+  Error.captureStackTrace(this, this.constructor)
+
+  this.message = [
+    `${dependency} does not have the correct version installed.`,
```

Comment:
This now requires ES6 string template support, are you sure you don't want to use regular string literals to make this module compatible with older versions of node?